app/views/students/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Students List</h1>
+
+<table>
+<thead>
+  <tr>
+    <th>First Name</th>
+    <th>Last Name</th>
+    <th>Email address</th>
+    <th>Actions</th>
+  </tr>
+</thead>
+<tbody>
+  <% @students.each do |student| %>
+    <tr>
+      <th><%= student.first_name %></th>
+      <th><%= student.last_name %></th>
+      <th><%= student.email %></th>
+      <th><%= link_to 'show', student_path(student)  %></th>
+      <th><%= link_to 'edit', edit_student_path(student)  %></th>
+      <th><%= link_to 'delete', student_path(student), data: {
+        turbo_method: :delete,
+        turbo_confirm: 'Are you sure?'
+      }  %></th>
+    </tr>
+  <% end %>
+</tbody>
+</table>
```

Comment:
Hi @tsheporamantso 
Great work on showing a list of all students and also adding link to helpers to show, edit and delete student. Kindly also add another link to helper for student creation.